src/test/java/uk/ac/cam/cl/dtg/segue/auth/SegueLocalAuthenticatorTest.java

```diff
@@ -68,7 +68,7 @@ public class SegueLocalAuthenticatorTest {
 	 * @throws Exception
 	 *             - test exception
 	 */
-	@Before
+	@BeforeEach
```

Comment:
Before is the JUnit4 equivalent to BeforeEach, there shouldn't be any changes in behaviour here. If you want I could go and check if it would work with BeforeAll?

Reply:
Okay, looks like changing that to BeforeAll does indeed break several tests, so this is staying as BeforeEach